reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -319,21 +319,29 @@ A cast can also be performed when a variable is assigned to using [cast notation
 
 #### Comma operator `,`
 
-As a binary operator, the comma creates an array. As a unary operator, the
-comma creates an array with one member. Place the comma before the member.
+As a binary operator, the comma creates an array or appends to the array being
+created. In expression mode, as a unary operator, the comma creates an array
+with just one member. Place the comma before the member.
```

Comment:
```suggestion
As a binary operator in an expression, the comma creates an array or appends to the array being created. As a unary operator, placed before a member, the expression creates an array with just one member. Since `Write-Object` expects an argument, you must put the expression in parentheses.
```